core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/DefaultRealmService.java

```diff
@@ -374,8 +372,7 @@ public MultiTenantRealmConfigBuilder getMultiTenantRealmConfigBuilder()
 	}
 
 	private void errorEncountered(Exception e) throws UserStoreException {
-		String msg = "Exception while creating multi tenant builder " + e.getMessage();
-		log.error(msg, e);
+		String msg = "Exception while creating multi tenant builder ";
```

Comment:
We can remove this String definition and use the String value directly, as it's used only once.